README.md

```diff
@@ -50,8 +51,57 @@ those on using environment variables
 
     PORT=3001 DEBUG=server webdriverio-server
 
+### Running the webdriverio-server on your localhost
+
+1. First install [Graphics Magick](http://www.graphicsmagick.org/README.html). It is a critical dependency for screenshot processing.
+2. Next, fork and clone the webdriverio-server code from the ciena-blueplanet repository and make any changes.
+3. If you are not already in the top level of the webdriverio-server directory, go there (cd webdriverio-server)
+4. Once you have made satisfactory changes or you just want to run the server on your localhost, download a project that utilizes e2e tests (see below for examples on using the server)
+5. Run ```npm start``` on the webdriverio-server code
+6. If you are running a project that is already configured to run e2e tests, you can run those now. If not, configure your project 
+using the (webdriverio-client)[https://github.com/ciena-blueplanet/webdriverio-client] package. Instructions on configuring your project to run e2e tests can 
+be found in the README.md in the webdriverio-client repo on GitHub
+
+
 ## Using the server
 
 The best way to use the server is to utilize the [webdriverio-client](https://github.com/ciena-blueplanet/webdriverio-client) to make test requests.
 
 For an example usage of the client, take a look at [ember-frost-checkbox](https://github.com/ciena-frost/ember-frost-checkbox.git)
+
+# Running the Front End Application
+
+    cd webdriverio-app
+    npm install
+    npm start
+
+Navigate to localhost:4200 to view the current version of the website
+
+# Frequently Asked Questions
+
+#### Where is the important information stored?
+
+The wdio server contains a secure redis db accessed through a small express application 
+where the authorized usernames and corresponding testing tokens are stored.
+
+#### What sensitive computing resources are trying to be protected? 
+We are trying to prevent people from breaking the wdio server since anyone can submit node scripts to the server. 
```

Comment:
I think "breaking into the wdio server" instead of "breaking the wdio server," although both are technically correct.